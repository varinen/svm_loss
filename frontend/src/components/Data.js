import React from "react";
import {Card, CardHeader, CardContent} from "@material-ui/core";

import useStyles from "../styles";
import Article from "./Data/Article";

const Data = () => {
    const classes = useStyles();
    return (
        <Card className={classes.card}>
            <CardHeader title="Data Point And Loss Values">
            </CardHeader>
            <CardContent>
                <Article/>
            </CardContent>
        </Card>
    )
};

export default Data;