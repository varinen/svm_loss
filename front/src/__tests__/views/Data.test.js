import React from 'react';
import {shallow} from 'enzyme';
import Data from '../../views/SVMLoss/Demo/Data';

describe('Data', () => {
    const component = shallow(<Data/>);

    it('renders correctly', () => {
        expect(component).toMatchSnapshot();
    });
});

