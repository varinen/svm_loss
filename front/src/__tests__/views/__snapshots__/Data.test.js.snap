// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Data renders correctly 1`] = `
<Card>
  <CardHeader
    color="info"
  >
    Data Point And Loss Values
  </CardHeader>
  <CardBody
    className="makeStyles-pt2-2"
  >
    <_default />
  </CardBody>
</Card>
`;
